pkg/forward/setup.go

```diff
@@ -99,32 +99,21 @@ func (f *Forwarder) setupAttackerCapture(bind netip.Addr) error {
 	}
 	f.iface = iface
 
-	handle, err := pcap.NewInactiveHandle(iface.Name)
+	nf, err := nflog.Open(&nflog.Config{
+		Group:    833,
```

Comment:
Does this number have any special meaning (apart from being the same in nft and here)? 

It would be great if we somehow documented this.